scheduler/clients/javascript/lib/helpers/credentials.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Partial credentials to be used for the client
+ */
+export type PartialCredentials = {
+  /**
+   * API key (admin)
+   */
+  apiKey?: string
+  /**
+   * Nettu account id (admin)
+   */
+  nettuAccount?: string
```

Comment:
Nettu or Nettei?

Reply:
For the moment `nettu` is still used everywhere
I think I'll just make one big PR renaming everything to nittei